flaskr/app.py

```diff
@@ -81,7 +81,9 @@ class MethodResult(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     method_id = db.Column(db.Integer, db.ForeignKey('methods.id'))
     search_result_id = db.Column(db.Integer, db.ForeignKey('search_results.id'))
-    relevance_rating = db.Column(db.Float)
+    relevance_rating_desciption = db.Column(db.Float)
+    relevance_rating_title = db.Column(db.Float)
+    weighted_relevancy_rating = db.Column(db.Float)
```

Comment:
I like having this stored in the DB. In the future it would be good to give the team a heads up before changing column names in the DB; this is what broke the GraphQL endpoint test, our query was looking for a column by that name.